### DISCUSSION TITLE
Linux runtimes

### DISCUSSION THREAD

**Discussion by stefano-elysium** [airsdk/Adobe-Runtime-Support#1853] [Q&A]
Dear team,

I don't want to use the SDK, I just want to run an .air file under linux. Where can I find the runtimes? 

Best regards

**Comment by ajwfrost** [comment]
Hi

There aren't any shared runtimes available for Linux, it's necessary to use the SDK to create a "bundled" application where the runtime can then be distributed alongside the application itself. So you would need to contact whoever provided you with the .air file (or use the AIR SDK yourself) to turn this into the required package.

thanks
